app/src/devTool/panel.tsx

```diff
@@ -1,24 +1,28 @@
 import * as React from 'react';
 import get from 'lodash/get';
 import { Control, useForm } from 'react-hook-form';
+import { useStateMachine } from 'little-state-machine';
 import { useEffect } from 'react';
 import colors from './colors';
 import PanelTable from './panelTable';
 import FormStateTable from './formStateTable';
 import { Button, Input } from './styled';
+import { setCollapse } from './settingAction';
 
 export default ({
   control: { fieldsRef, getValues, formState, errorsRef, readFormStateRef },
 }: {
   control: Control;
 }) => {
+  const { state, action } = useStateMachine(setCollapse);
   const [, setData] = React.useState({});
-  const [collapseAll, setCollapseAll] = React.useState(true);
   const [showFormState, setShowFormState] = React.useState(false);
   const fieldsValues = getValues();
   const { register, watch } = useForm();
   const searchTerm = watch('search', '');
 
+  console.log(state)
```

Comment:
Was this deliberate or forgotten? Could get noisy over time :)

Reply:
Shit! I will patch that.